metricshub-internaldb-extension/src/main/java/org/sentrysoftware/metricshub/extension/internal/db/SqlClientExecutor.java

```diff
@@ -1,8 +1,8 @@
-package org.sentrysoftware.metricshub.extension.local.sql;
+package org.sentrysoftware.metricshub.extension.internal.db;
```

Comment:
Not sure we need a package for `internal`.

```suggestion
package org.sentrysoftware.metricshub.extension.internaldb;
```